css/reaction.css

```diff
@@ -234,7 +234,7 @@ fieldset {
 #login:hover {
   border-color: black;
 }
-#sections td:first-child {
```

Comment:
Is this ID still referenced? If not, should we delete it?

Reply:
Yes, it's referenced in `reaction.js`.